cfg/aws/credentials.go

```diff
@@ -88,6 +88,12 @@ func getSession(config *aws.Config) *session.Session {
 		}
 	}
 	log.Printf("D! Successfully created credential sessions\n")
+	cred, err := ses.Config.Credentials.Get()
+	if err != nil {
+		log.Printf("E! Failed to get credential from session: %v", err)
+	} else {
+		log.Printf("D! Using credential %s from %s", cred.AccessKeyID, cred.ProviderName)
```

Comment:
**Questions:** 
* I am not confident if we are allowed to log sensitive credential (without encryption) in `amazon-cloudwatch-agent.log`
* What would happen for CW Internal Agent though?

Reply:
You need both the access key ID and secret key to use it. Also the AWS SDK logs the access key ID for requests:
```
Authorization: AWS4-HMAC-SHA256 Credential=ASIAT6CRPX66DBNTTAWB/20220712/us-east-1/ec2/aws4_request
```

What does this have to do with internal agent

Reply:
> Also the AWS SDK logs the access key ID for requests:

Interesting.

> What does this have to do with internal agent

Internal Agent does use similar credential file mechanism or role ARN so want to ensure there are no errors when retrieve from Apo environment